src/box/lua/schema.lua

```diff
@@ -334,6 +334,9 @@ local function update_param_table(table, defaults)
     return new_table
 end
 
+box.utils = {};
+box.utils.check_param_table = check_param_table
```

Comment:
I don't think we need to make this helper public, because
- Nobody asked for it AFAIK.
- Making it a part of the public API means that we take on a burden of maintaining it and ensuring backward compatibility, which limits our freedom when it comes to code refactoring.

I think we should put it in `box.internal`, like we did with `check_space_arg`:

https://github.com/tarantool/tarantool/blob/8641990b9419c8ac89c5242465d11b2e5ffc2246/src/box/lua/schema.lua#L1790